src/JavascriptTrait.php

```
@@ -0,0 +1,344 @@
+        $message_parts[] = sprintf(
+          "  - Error: %s\n    Source: %s:%d",
+          $error_message,
+          $error_source,
+          $error_line
+        );
+      }
+    }
+
+    $message_parts[] = sprintf("\nTotal errors: %d", $error_count);
+
+    throw new \Exception(implode("\n", $message_parts));
```

PHPMD (2.15.0)

Missing class import via use statement (line '322', column '15'). (undefined)

(MissingImport)